tests/loaddata/test_digits_access.py

```diff
@@ -51,3 +53,38 @@ def test_get_train_test(dataset_name, download_path):
     assert source.n_classes() == 10
     assert isinstance(source_train, torch.utils.data.Dataset)
     assert isinstance(source_test, torch.utils.data.Dataset)
+
+
+@pytest.mark.parametrize("dataset_name", ALL)
```

Comment:
It may not be necessary to loop through all variations, if they were tested elsewhere.  We can talk tomorrow.